rhba-dev-env/ansible/group_vars/all/vars_file.yml

```diff
@@ -0,0 +1,59 @@
+---
+### environment vars ###
+workspace_home:  /home/vagrant/workspace
+# https://github.com/ansible/ansible/issues/54855
+ansible_python_interpreter: /usr/bin/python3
+# same "ip" from config.yml (Vagrantfile on vagrant-ocp-bsig)
+vm_ip: 192.168.0.40
+# same "name" from config.yml (Vagrantfile)
+vm_name: workstation
+
+### kerberos config ###
+kerberos_username: mmagnani
+kerberos_password: my_kerberos_password
+
+### useful scripts ###
+build_overrides_version: '7.4.0'
+build_overrides_product: 'rhpam'
+build_overrides_nightly: '20190724'
+build_product: 'rhpam-all'
+
+git_branch_target: remotes/origin/7.4.x
+
+### rhpam product vars - don't change  ###
+rhpam_7_openshift_image_repo_dir: rhpam-7-openshift-image
+rhpam_7_openshift_image_smartrouter_dir: smartrouter
+rhpam_7_openshift_image_businesscentral_dir: businesscentral
+rhpam_7_openshift_image_businesscentral_indexing_dir: businesscentral-indexing
+rhpam_7_openshift_image_businesscentral_monitoring_dir: businesscentral-monitoring
+rhpam_7_openshift_image_controller_dir: controller
+rhpam_7_openshift_image_kieserver_dir: kieserver
+
+### rhdm product vars - don't change  ###
+rhdm_7_openshift_image_repo_dir: rhdm-7-openshift-image
+rhdm_7_openshift_image_decisioncentral_dir: decisioncentral
+rhdm_7_openshift_image_decisioncentral_indexing_dir: decisioncentral-indexing
+rhdm_7_openshift_image_controller_dir: controller
+rhdm_7_openshift_image_kieserver_dir: kieserver
+rhdm_7_openshift_image_optaweb_employee_rostering_dir: optaweb-employee-rostering
+
+### my images override ###
+my_images_overrides_file: my-images-overrides.yaml
+# cct_module
+cct_module_repo: https://github.com/jboss-openshift/cct_module.git
+cct_module_repo_ref: master
+# jboss-eap-modules
+jboss_eap_modules_repo: https://github.com/jboss-container-images/jboss-eap-modules.git
+jboss_eap_modules_ref: master
+# jboss-eap-7-image
+jboss_eap_7_image_repo: https://github.com/jboss-container-images/jboss-eap-7-image.git
+jboss_eap_7_image_ref: eap72
```

Comment:
I think the branches are outdated.

Reply:
Correct.

Reply:
This conversation was marked as Resolved, however the value still says "eap72". If you are looking for what the equivalent of "master" is in that repo, I think it is technically eap72-dev on github, however I believe that 7.2.x is actually what they are using for it.